notebooks/synthetic_test_spatial_disaggregation.ipynb

```diff
@@ -0,0 +1,2507 @@
+{
```

Comment:
<p>I think you were already thinking something like this, but in the scaled up implementation, I think you could do something along the lines of what's done in QDM adjustment <a href="https://github.com/ClimateImpactLab/xclim/blob/410a3c28e5688d6372db3e53cc2da6937f9dd926/xclim/sdba/adjustment.py#L402" target="_blank">here</a> instead of looping, finding indices, and appending. (Not sure though, haven't thought it all the way through).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ClimateImpactLab/downscaleCMIP6/pull/88/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>